Sources/Filters/General/AppendPolyData/index.js

```diff
@@ -0,0 +1,238 @@
+import macro            from 'vtk.js/Sources/macro';
+import vtkDataArray     from 'vtk.js/Sources/Common/Core/DataArray';
+import vtkPoints        from 'vtk.js/Sources/Common/Core/Points';
+import vtkPolyData      from 'vtk.js/Sources/Common/DataModel/PolyData';
+import { VtkDataTypes } from 'vtk.js/Sources/Common/Core/DataArray/Constants';
+
+const { vtkErrorMacro } = macro;
+
+export const PointPrecision = {
```

Comment:
Those constants should be part of a constant file. 

See https://github.com/Kitware/vtk-js/tree/master/Sources/Common/Core/DataArray for example